src/app/invest/invest.component.scss

```diff
@@ -29,42 +29,22 @@ p {
   &__input-label p {
     text-transform: uppercase;
     color: $title-color;
-    display: none;
   }
 
   &__text-right {
-    display: none;
-    align-items: center;
-    justify-content: space-between;
-    margin-top: 20px;
-
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    padding: 24px 20px;
+    z-index: 1001;
+    background-color: $secondary-color;
+    border-top: 1px solid $background-color;
```

Comment:
CSS for new mobile funds section

Reply:
I like that look with fixed position and z-index on small devices 👍